README.md

```diff
@@ -102,6 +102,12 @@ data class Post(
 
 ```
 
+<h4>Polymorphic serialization (sealed classes)</h4>
+
+This sdk will handle polymorphic serialization automatically if you have a sealed class and its children marked as `Serializable`. It will include a `type` property that will be used to discriminate which child class is the serialized.
+
+You can change this `type` property by using the `@FirebaseClassDiscrminator` annotation in the parent sealed class.
+
```

Comment:
example of using the annotation would be good here

Reply:
Done! :)